themes/stanford-style/lms/templates/register-form.html

```diff
@@ -26,7 +26,14 @@ <h3 class="message-title">${_("The following errors occurred while processing yo
 
   % for enabled in provider.Registry.accepting_logins():
     ## Translators: provider_name is the name of an external, third-party user authentication service (like Google or LinkedIn).
-    <button type="submit" class="button button-primary button-${enabled.provider_id} register-${enabled.provider_id}" onclick="thirdPartySignin(event, '${pipeline_urls[enabled.provider_id]}');"><span class="icon fa ${enabled.icon_class}"></span>${_('Sign up with {provider_name}').format(provider_name=enabled.name)}</button>
+    <button type="submit" class="button button-primary button-${enabled.provider_id} register-${enabled.provider_id}" onclick="thirdPartySignin(event, '${pipeline_urls[enabled.provider_id]}');">
+        % if enabled.icon_class:
+        <span class="icon fa ${enabled.icon_class}" aria-hidden="true"></span>
+        % else:
+        <img class="icon" src="${enabled.icon_image.url}" aria-hidden="true" />
```

Comment:
`<img` tags should always have an `alt` attribute; since we don't want screen readers to read anything here, we've marked it as aria-hidden but you should still put `alt=""`